Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'capistrano-rails', group: :development
 
 gem 'newrelic_rpm'
+gem 'mini_portile2', '~> 2.5', '>= 2.5.1'
```

Comment:
heroku build was suddenly failing, erroring out from missing this gem. as a workaround we are explicitly adding it